### PR TITLE
Fix the path issue of `install.ps1`

### PR DESCRIPTION
**Pull request by pk5ls20**
Try to resolve https://github.com/mamba-org/micromamba-releases/issues/40

### CODE REVIEW

File: install.ps1 [hunk 1]
```diff
@@ -38,7 +38,7 @@ if ($choice -eq "y" -or $choice -eq "Y" -or $choice -eq "") {
     Write-Output "Initializing micromamba in  $prefix"
     $MAMBA_INSTALL_PATH = Join-Path -Path $Env:LocalAppData -ChildPAth micromamba\micromamba.exe
```
Comment (teleksterling) [inline install.ps1:39]:
```suggestion
    $MAMBA_INSTALL_PATH = Join-Path -Path $Env:LocalAppData -ChildPath micromamba\micromamba.exe
```

File: install.ps1 [hunk 2]
```diff
@@ -38,7 +38,7 @@ if ($choice -eq "y" -or $choice -eq "Y" -or $choice -eq "") {
...
     Write-Output $MAMBA_INSTALL_PATH
-    & $MAMBA_INSTALL_PATH shell init -s powershell -p $Env:UserProfile\micromamba
+    & $MAMBA_INSTALL_PATH shell init -s powershell -p $prefix
 } else {
     Write-Output "`nYou can always initialize powershell pr cmd.exe with micromamba by running `nmicromamba shell init -s powershell -p $Env:UserProfile\micromamba`n"
```
Comment (teleksterling) [inline install.ps1:43]:
```suggestion
    Write-Output "`nYou can always initialize powershell or cmd.exe with micromamba by running `nmicromamba shell init -s powershell -p $Env:UserProfile\micromamba`n"
```

**Review by teleksterling — APPROVED** [review]
I found the same issue (and solution). So I can confirm your changes work for my installation too.
I've suggested a couple other typos to fix in the surrounding code.

**Comment by JohanMabille** [comment]
Thanks!

**Pull request merged**